src/components/calcite-action-bar/readme.md

```diff
@@ -4,8 +4,9 @@
 
 The `calcite-action-bar` component is made up of multiple `calcite-actions` in the form of clickable icons. The action bar can be expanded to view actions with descriptive text or made smaller to view with just icons.
 
-<!-- Auto Generated Below -->
+See the demo [here](https://esri.github.io/calcite-app-components/demos/calcite-action-bar.html).
```

Comment:
Can we link the word “demo” instead of “here”? This applies other places in the PR.

Reply:
do you want me to change the link in the readmes as well? they follow the same pattern

Reply:
i mean the link TO the readmes in the demos

Reply:
Yeah 👍 

For accessibility and readability, text should describe the link location so `See the [action-bar demo]` would be better.